apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx

```diff
@@ -0,0 +1,45 @@
+import { BranchData } from 'data/branches/branch-query'
+import { AlertTriangleIcon, ClockIcon } from 'lucide-react'
+import { Badge } from 'ui'
+
+type Status = BranchData['status']
+
+export interface BranchStatusBadgeProps {
+  status: Status
+}
+
+const UNHEALTHY_STATUES: Status[] = ['ACTIVE_UNHEALTHY', 'INIT_FAILED', 'UNKNOWN']
+const WAITING_STATUSES: Status[] = ['COMING_UP', 'GOING_DOWN', 'PAUSING', 'RESTORING', 'UPGRADING']
+
+const STATUS_TO_LABEL: Record<Status, string> = {
+  ACTIVE_HEALTHY: 'Healthy',
+  ACTIVE_UNHEALTHY: 'Unhealthy',
+  INIT_FAILED: 'Init failed',
+  UNKNOWN: 'Unknown',
+  COMING_UP: 'Coming up',
+  GOING_DOWN: 'Going down',
+  INACTIVE: 'Inactive',
+  PAUSING: 'Pausing',
+  REMOVED: 'Removed',
+  RESTORING: 'Restoring',
+  UPGRADING: 'Upgrading',
+}
+
+const BranchStatusBadge = ({ status }: BranchStatusBadgeProps) => {
+  if (status === 'ACTIVE_HEALTHY') {
+    return null
+  }
+
+  const isUnhealthy = UNHEALTHY_STATUES.includes(status)
+  const isWaiting = WAITING_STATUSES.includes(status)
+
+  return (
+    <Badge color={isUnhealthy ? 'red' : 'slate'} className="flex items-center gap-1.5">
+      {isUnhealthy && <AlertTriangleIcon size={12} />}
```

Comment:
use heroicon ?

Reply:
<img width="423" alt="Screenshot 2024-01-22 at 10 15 33 PM" src="https://github.com/supabase/supabase/assets/8291514/2003ae80-a649-4c04-832b-1c635bd0fc20">

Reply:
We already use this in Docs for callouts

Reply:
do you has link pls?

Reply:
i created one in NotificationsPopover.constants.ts - maybe lets just bring that out into components/ui 
either that or we bring it into the UI library (which sounds better imo)

unless docs is already using a component from the ui library (but last i checked i saw that it was just SVG)